File: agent-api/app/services/chat/builtin_assistants/interview/definition.py
"""Stable interview identity and catalog defaults without runtime imports."""

from app.services.chat.builtin_assistants.types import BuiltinAppSpec, BuiltinAssistantDefinition

INTERVIEW_PRESET = "interview"
INTERVIEW_APP_ID = "builtin:interview"
INTERVIEW_THREAD_ORIGIN = "interview"
INTERVIEW_IDENTITY = {
    "app_id": INTERVIEW_APP_ID, "preset": INTERVIEW_PRESET, "name": "面试助手",
    "origin": INTERVIEW_THREAD_ORIGIN, "ui_policy_key": "interview_practice",
}
INTERVIEW_DEFINITION = BuiltinAssistantDefinition(
    identity=INTERVIEW_IDENTITY,
    catalog=BuiltinAppSpec(
        preset=INTERVIEW_PRESET, name="面试助手",
        description="结合简历与岗位 JD，进行文字模拟面试、逐题反馈与复盘",
        icon="", category="communication", category_label="沟通交互",
        route="/center/chat/interview", order_num=30,
    ),
)


def is_interview_preset(value: object) -> bool:
    return str(value or "").strip().lower() == INTERVIEW_PRESET

File: src/views/peopleCenter/builtinAssistants/interview/definition.ts
import { INTERVIEW_ASSISTANT_PRESET, type BuiltinAssistant } from '../types';

export const INTERVIEW_ASSISTANT_ID = 'builtin:interview';

export const INTERVIEW_ASSISTANT: BuiltinAssistant = {
  appId: INTERVIEW_ASSISTANT_ID,
  preset: INTERVIEW_ASSISTANT_PRESET,
  name: '面试助手',
  shortBadge: '面试助手',
  description: '根据简历与目标岗位进行文字模拟面试，逐答反馈并复盘提升',
  icon: '/agent-icons/builtin/interview-assistant-v3.svg',
  legacyIcon: '/agent-icons/work-agent-orb.svg',
  category: 'communication',
  categoryLabel: '沟通交互',
  route: '/center/chat/interview',
  entryMode: 'standalone',
  recommendOrder: 3,
  mascotVariant: 'interview',
  uiPolicy: 'interview_practice',
  notices: {
    useSkill: '面试练习使用当前简历与岗位材料，其他 Skill 请在主对话中使用',
    useKnowledge: '面试考察围绕本场目标岗位展开，请在开始前填写岗位要求',
  },
};
